calico-cloud/image-assurance/scanners/cli-based-scanner.mdx

```diff
@@ -0,0 +1,195 @@
+---
+description: Identify vulnerabilities in workloads to Kubernetes clusters.
+---
+
+# Manually Scan images for vulnerabilities using CLI
+
+## Big picture
+
+Scan images for vulnerabilities using {{prodname}} Image Assurance.
+
+## Value
+
+{{prodname}} Image Assurances helps you identify vulnerabilities in workloads that you deploy to Kubernetes clusters. A vulnerability can be a weakness in an application (for example, a design flaw or an implementation bug) that allows attackers to cause additional harm. With Image Assurance you can:
+
+- Scan an image for vulnerabilities
+- Assess the impact of newly-found vulnerabilities and prioritize remediation efforts
+- Create exceptions to ignore specific vulnerabilities
+- Create alerts on high-severity vulnerabilities so you can delegate remediation efforts to the appropriate team
+- Block non-compliant workloads using policy as part of your cloud-native security posture
```

Comment:
If this boiler plate intro section is common amongst the pages, shouldn't we provide links to the approach page / section so that the user can jump to that part in the docs?

Reply:
I added an `overview` section for the boiler plate PTAL cc: @bmckercher123